src/app/participant-list/participant-list.component.ts

```diff
@@ -1281,12 +1353,16 @@ export class ParticipantListComponent implements OnInit {
       this.participantList.sort( ( a, b ) => ( a.data == null || b.data == null ) ? 1 : this.sort( a.data, b.data, order, this.sortField, colType ) );
     } else if (this.sortParent === "p") {
       this.participantList.sort( ( a, b ) => {
-        if (a.participant == null || a.participant[ this.sortField ] == null) {
+        if (a.participant == null || (a.participant[ this.sortField ] == null && a.participant['additionalValues'] == null)) {
```

Comment:
which one is the right version to move into angular10 without performance changes